File: AgentWebX5/src/main/java/com/just/agentwebX5/JsInterfaceHolder.java
package com.just.agentwebX5;

import androidx.collection.ArrayMap;

/**
 * Created by cenxiaozhong on 2017/5/13.
 * source CODE  https://github.com/Justson/AgentWebX5
 */

public interface JsInterfaceHolder {

    JsInterfaceHolder addJavaObjects(ArrayMap<String, Object> maps);

    JsInterfaceHolder addJavaObject(String k, Object v);

    boolean checkObject(Object v) ;

}

File: scaffold/src/main/java/top/xuqingquan/web/agent/JsInterfaceHolder.kt
package top.xuqingquan.web.agent

interface JsInterfaceHolder {

    fun addJavaObjects(maps: Map<String, Any>): JsInterfaceHolder

    fun addJavaObject(k: String, v: Any): JsInterfaceHolder

    fun checkObject(v: Any): Boolean

}
